tests/Doctrine/Tests/ORM/Functional/Locking/LockTest.php

```diff
@@ -35,13 +37,16 @@ public function testLockVersionedEntity() {
         $this->_em->flush();
 
         $this->_em->lock($article, LockMode::OPTIMISTIC, $article->version);
+
+        $this->addToAssertionCount(1);
```

Comment:
How would we do that? I mean, optimistic locks are done based on the entity version and they raise an exception when versions don't match.